plugins/netscaler-k8s.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: netscaler-k8s
+spec:
+  shortDescription: A Kubernetes plugin for inspecting Ingress Controller and associated NetScaler deployments
+  description: |
+    The official kubectl plugin for netscaler-k8s.
```

Comment:
this is where you add the useful information about what the plugin actually does. consider using it.

Reply:
Fixed in latest commit